lib/rubocop/cop/rspec/align_left_let_brace.rb

```diff
@@ -40,6 +37,16 @@ def on_new_investigation
             end
           end
         end
+
+        private
+
+        def token_aligner
+          RuboCop::RSpec::AlignLetBrace.new(
+            processed_source.ast,
+            :begin,
+            rspec_language_config
```

Comment:
Maybe push `rspec_language_config` to `AlignLetBrace` not to pass it?

Reply:
This seems impossible as well.
We may rework `AlignLetBrace` to be an abstract base class for those two, but it's completely outside of the scope.